UKHomeOffice/nmsw-ui
# NMSW-5 Add favicon

# Ticket

NMSW-5

----

## AC

Add favicon


----

## To test

- run app locally
- go to localhost:3000 in Edge
- > the tab should show the GovUK Icon
- go to localhost:3000 in Firefox
- > the tab should show the GovUK Icon
- go to localhost:3000 in Safari
- > the tab should show the GovUK Icon (inverted colour version)

----

## Notes

Chrome will not show the favicon in local mode so we cannot test that yet.
We can't run this code on mobile devices to test them yet.
NMSW-39 has been created so we can do cross browser testing once the dev environment is setup and this is deployed to it.
